packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -242,20 +257,81 @@ class DDSFooterComposite extends ModalRenderMixin(
     `;
   }
 
-  renderLightDOM() {
+  renderLanguageSelector(slot = 'language-selector') {
     const {
-      buttonLabel,
       clearSelectionLabel,
-      disableLocaleButton,
-      langDisplay,
+      _isMobile: isMobile,
       langList,
       languageSelectorLabel,
       selectedLanguage,
       size,
+    } = this;
+    return isMobile
+      ? html`
+          <dds-language-selector-mobile
+            size="${size || DROPDOWN_SIZE.EXTRA_LARGE}"
+            slot="${slot}"
+            value="${selectedLanguage}"
+            placeholder="${selectedLanguage}">
+            ${langList?.map(
+              (language) => html`
+                <bx-select-item
+                  label="${ifNonNull(language.text)}"
+                  value="${ifNonNull(language.text)}"
+                  lang="${ifNonNull(language.id)}"
+                  >${ifNonNull(language.text)}</bx-select-item
+                >
+              `
+            )}
+          </dds-language-selector-mobile>
+        `
+      : html`
+          <dds-language-selector-desktop
+            size="${size || DROPDOWN_SIZE.EXTRA_LARGE}"
```

Comment:
same here: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/fcaa459c82b197b760e589c1378e4b3c1041058b/packages/web-components/src/components/footer/language-selector-desktop.ts#L131
```suggestion
```